tests/pydantic/test_basic_types_pyd.py

```diff
@@ -45,6 +44,8 @@ def test_fhirdate():
     model = ExampleModel(date="1973-06")
     model = ExampleModel(date="1905-08-23")
 
+    del model
```

Comment:
Why do we need this delete?

Reply:
Hmm, not necessary indeed. The delete is purely artificial to use the `model` variable at least once (`F841 local variable 'model' is assigned to but never used`)

Reply:
then I would rather disable the qa here than to change something semantically